packages/block-library/src/search/style.native.scss

```diff
@@ -11,31 +11,83 @@
 }
 
 .border {
-	border: 1px solid $gray-600;
-	border-width: $border-width;
-	padding: $grid-unit-05;
-}
-
-.searchTextInput {
-	flex-grow: 1;
-	flex-shrink: 1;
-	padding: $grid-unit-05;
+	border: $border-width solid $gray-5;
+	border-radius: $grid-unit-05;
+	padding-left: $grid-unit-10;
```

Comment:
We could rewrite this to 
```
padding-left: $grid-unit-10;
padding-right: $grid-unit-05;
padding-top: $grid-unit-05;
padding-bottom: $grid-unit-05;
```
as 
```
padding: $grid-unit-05;
padding-left: $grid-unit-10;
```

Reply:
Thanks for the tip! Simplified many of the style options in 7c9a6dd68d4d2d9d0a3d8e391c59982e34ff59bc